docs/tutorials/blog.rst

```diff
@@ -14,25 +18,108 @@ Prerequisites:
 
  - :doc:`Install Elgg</intro/install>`
 
+Create the plugin's directory and manifest file
+===============================================
+
+First, choose a simple and descriptive name for your plugin.
+In this tutorial, the name will be ``my_blog``.
+Then, create a directory for your plugin in the ``/mod/`` directory
+found in your Elgg installation directory. Other plugins are also located
+in ``/mod/``. In this case, the name of the directory should
+be ``/mod/my_blog/``. This directory is the root of your plugin and all the
+files that you create for the new plugin will go somewhere under it.
+
+Next, in the root of the plugin, create the plugin's manifest file,
+``manifest.xml``:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
+        <name>My Blog</name>
+        <id>my_blog</id>
+        <author>Your Name Here</author>
+        <version>0.1</version>
+        <description>Adds blogging capabilities.</description>
+        <requires>
+            <type>elgg_release</type>
+            <version>2.0</version>
+        </requires>
+    </plugin_manifest>
+
+See :doc:`Plugins</guides/plugins>` for more information
+about the manifest file.
+
+Create the form for creating a new blog post
+================================================
+
+Create a file at ``/mod/my_blog/views/default/forms/my_blog/save.php``
+that contains the form body.
+The form should have input fields for the title, body and tags
+of the my_blog post. It does not need form tag markup.
+
+.. code-block:: php
+
+    <?php
+
+    // creates a field where you can write the blog post's title
+    echo elgg_view_input('text', [
+        'name' => 'title',
+        'label' => 'Title of your post',
```

Comment:
This should be:
```
'label' => elgg_echo('my_blog:title'),
```
so that new plugin devs immediately learn how to create translatable user interfaces.

Or do you think it is better to keep the examples more simple (this is after all only a basic tutorial)?